crates/languages/src/go/runnables.scm

```diff
@@ -0,0 +1,23 @@
+; Functions names start with `Test`
+(
+    (function_declaration name: (_) @run
+        (#match? @run "^Test.+"))
+) @go-test
```

Comment:
Readability nit: could we use a separate name for the thing you're matching on and run position itself?

```suggestion
(
    (function_declaration name: (_) @run @_name
        (#match? @_name "^Test.+"))
) @go-test
```
Same goes for other queries in this file.